packages/perseus/src/widgets/dropdown/prompt-utils.test.ts

```diff
@@ -0,0 +1,29 @@
+import {WidgetType} from "../../prompt-types";
+
+import {getPromptJSON} from "./prompt-utils";
+
+import type {PerseusDropdownUserInput} from "../../validation.types";
+
+describe("Dropdown getPromptJSON", () => {
+    it("it returns JSON with the expected format and fields", () => {
+        const renderProps: any = {
+            choices: ["Pickles", "Tomato", "Onion", "Lettuce"],
+        };
+
+        const userInput: PerseusDropdownUserInput = {
+            value: 1,
+        };
+
+        const resultJSON = getPromptJSON(renderProps, userInput);
+
+        expect(resultJSON).toEqual({
+            type: WidgetType.DROPDOWN,
+            options: {
+                items: renderProps.choices,
+            },
+            userInput: {
+                selectedIndex: userInput.value - 1,
```

Comment:
I liked your comment on the other one
```suggestion
                // Offset to account for placeholder
                selectedIndex: userInput.value - 1,
```